workspaces/jenkins/plugins/jenkins-backend/src/service/jenkinsInfoProvider.ts

```diff
@@ -291,4 +317,24 @@ export class DefaultJenkinsInfoProvider implements JenkinsInfoProvider {
       ]
     );
   }
+
+  private static getEntityOverrideURL(entity: Entity) {
+    return entity.metadata.annotations?.[
+      DefaultJenkinsInfoProvider.JENKINS_OVERRIDE_URL
+    ];
+  }
+
+  private static verifyUrlMatchesRegex(url: string, regexList: string[]) {
+    for (const regexString of regexList) {
+      try {
+        const regex = new RegExp(regexString);
+        if (regex.test(url)) {
+          return true;
+        }
+      } catch (e) {
+        console.error(`Invalid regex: "${regexString}" - Error: ${e.message}`);
```

Comment:
Sorry but one more thing; no console, use LoggerService instead

Reply:
@drodil just added the loggerService, and no worries
Just take an extra look as I had to add LoggerService to the default info provider